docs/keepalive.md

```diff
@@ -0,0 +1,2 @@
+# Keepalive
```

Comment:
I assume there are some plans to add to this?

Reply:
Yes, I was waiting to be done with the implementation first. Just a reminder for me to create some documentation before this gets merged.